### ISSUE TITLE
Lots of page fault errors in Win98SE on arm build

### ISSUE THREAD

**Issue by wn2000** [schellingb/dosbox-pure#288]
RetroArch version: 1.8.7
Dosbox Pure version: 0.9.6 (arm 32bit build)
Platform: Rockchip RK3328
Symptom:
After booting into Win98SE, there are frequent popup dialogs with "page fault" errors, from File Explorer, rundll, or just any Windows application, making the system not really usable.

**Comment by michaeljsiena** [comment]
Same (also reported with screenshots in issue 284).

**Comment by wn2000** [comment]
> Same (also reported with screenshots in issue #284).

Yep can confirm I got exactly those same errors during and after installation. You probably should update the title of #284 to reflect the new errors.

**Comment by Ultrahead** [comment]
Confirming that this also happens on a ARM64 Linux OS (Tina Linux) on the Trimui Smart Pro device
